src/testing_system/internal/adapter/assistants/ollama.py

```
@@ -0,0 +1,77 @@
+                 ):
+        self.model = model
+        self.base_url = base_url
+        self._ensure_model()
+    
+    def _ensure_model(self) -> None:
+        """Model checker"""
+        
+        try:
+            response = requests.get(f"{self.base_url}/api/tags", timeout=10)
+        except requests.ConnectionError:
+            raise RuntimeError(f"Ollama not available at {self.base_url}")
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)